index.js

```diff
@@ -39,14 +39,14 @@ export default function (window) {
     }
 
     options.scale = options.scale || 1
-    switch (options.style) {
-      case "scratch2":
-        return scratch2.newView(doc, options)
-      case "scratch3":
-        return scratch3.newView(doc, options)
-      default:
-        throw new Error(`Unknown style: ${options.style}`)
+
+    if (options.style.startsWith("scratch2")) {
+      return scratch2.newView(doc, options)
+    } else if (options.style.startsWith("scratch3")) {
```

Comment:
It's weird to me that this will match `scratch30`. Perhaps this?
```suggestion
    } else if (/^scratch3($|-)/.test(options.style)) {
```
Or partition on `-` and then do a plain `switch` on the first word?

Or just explicitly match on `scratch3-high-contrast` here--that's probably fine?